src/Binance/CoinMFutures/API/Ticker24hr.jl

```diff
@@ -0,0 +1,101 @@
+module Ticker24hr
```

Comment:
```suggestion
module Ticker
```

e.g. https://github.com/bhftbootcamp/CryptoAPIs.jl/blob/master/src/Binance/Spot/API/Ticker.jl